config/routes.rb

```diff
@@ -15,6 +15,7 @@
   namespace :user, path: ":user_name", as: :user do
     resources :properties, only: [:create, :index, :update, :show]
   end
+  resources :properties, only: [:index]
```

Comment:
you'll need to also include the ":new" in this since my previous PR already had a route similar to yours.